### DISCUSSION TITLE
"Starlette's Middleware docs" link on "Advanced Middleware" documentation is broken

### DISCUSSION THREAD

**Discussion by DavidBord** [fastapi/fastapi#8907] [Questions] [answered]
It links to https://www.starlette.io/middleware/ where it should link to https://www.starlette.io/middleware.html
because https://www.starlette.io/middleware/ gets 404 error

**Comment by Kludex** [comment]
Do we need to enable this https://www.mkdocs.org/user-guide/configuration/#use_directory_urls ?

EDIT: Wait, I'm in the FastAPI issues :sweat_smile: This is a starlette issue. :thinking: 

@DavidBord Do you mind filling an issue on [Starlette](https://github.com/encode/starlette)?

**Comment by ArcLightSlavik** [comment]
Think starlette docs destroyed themself #3402 #3402 and https://github.com/encode/starlette/issues/1212

**Comment by tomwojcik** [comment, marked as answer]
Fixed in StarletteFixed in Starlette
